### PR TITLE
[nl] Allow fulfillment when there are only multi svs detected

### PR DESCRIPTION
**Pull request by chejennifer**
When there are no svs detected because the scores were too low, we should still try to populate charts for multi sv candidates (if there are any detected)

This fixes the empty result in strict mode for the query 'Does obesity correlate with lack of sleep in US counties' (it returns results in regular mode).

Example: nodejs/query?q=Does%20obesity%20correlate%20with%20lack%20of%20sleep%20in%20US%20counties
currently: empty result
with this change: https://paste.googleplex.com/5581611641339904

### CODE REVIEW

File: server/lib/nl/fulfillment/base.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def populate_charts(state: PopulateState) -> bool:
   places = state.uttr.places
   handle_contained_in_type(state, places)
 
-  if not state.uttr.svs:
+  if not state.uttr.svs and not state.uttr.multi_svs:
```
Comment (pradh) [inline server/lib/nl/fulfillment/base.py:58]: Should we check `state.chart_vars_map` instead?

Also, do the same in line 68?

File: server/lib/nl/fulfillment/base.py [hunk 2]
```diff
@@ -55,7 +55,7 @@ def populate_charts(state: PopulateState) -> bool:
...
     state.uttr.counters.err('num_populate_fallbacks', 1)
     state.uttr.sv_source = FulfillmentResult.UNRECOGNIZED
     return False
```
Comment (pradh) [inline server/lib/nl/fulfillment/base.py:61]: Also, don't add `state.uttr.svs` in RHS line 67.

Also, drop RHS line 190+191 (printing state.uttr.svs to log)?

**Review by pradh — COMMENTED** [review]
Nice find!

File: server/lib/nl/fulfillment/base.py (hunk 1 above)
Comment (chejennifer) [reply server/lib/nl/fulfillment/base.py]: ah yes, that makes sense, updated

File: server/lib/nl/fulfillment/base.py (hunk 2 above)
Comment (chejennifer) [reply server/lib/nl/fulfillment/base.py:61]: done

File: server/lib/nl/fulfillment/base.py [hunk 3]
```diff
@@ -188,7 +189,7 @@ def _maybe_switch_parent_type(
 def _add_charts_with_existence_check(state: PopulateState,
                                      places: List[Place]) -> bool:
   svs = state.uttr.svs
```
Comment (pradh) [inline server/lib/nl/fulfillment/base.py:191]: Drop the logging (we don't really use it)?   Also is `svs` variable used?

**Review by pradh — APPROVED** [review]
Thanks!

File: server/lib/nl/fulfillment/base.py (hunk 3 above)
Comment (chejennifer) [reply server/lib/nl/fulfillment/base.py]: dropped! the `svs` variable was used later in line 251, but removed `svs` now and updated that line to just use `state.uttr.svs`

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**